### PR TITLE
chore: bump k8s/tekton client to 6.4.0 (#641)

### LINKED ISSUE #641: migrate to kubernetes-client 6.4.0
Hints and instructions are documented at https://github.com/fabric8io/kubernetes-client/blob/master/doc/MIGRATION-v6.md

### PR DESCRIPTION
**Pull request by lstocchi**
it resolves #641

### CODE REVIEW

**Comment by adietish** [comment]
@lstocchi add a commit that bumps kubernetes-client to `6.4.0` and intellij-common to `1.9.0-SNAPSHOT` (which was bumped to `6.4.0`)

File: build.gradle [hunk 1]
```diff
@@ -36,9 +36,9 @@ runIdeForUiTests {
 }
```
Comment (adietish) [inline build.gradle:36]: `build.gradle` needs to be updated to depend on the bumped intellij-kubernetes once its PR https://github.com/redhat-developer/intellij-kubernetes/pull/548 is merged
```
plugins = [...'com.redhat.devtools.intellij.kubernetes:0.7.1.XXX']
```

**Comment by robstryker** [comment]
This build seemed to work for me locally.

**Comment by adietish** [comment]
Build seems fine once you change `build.gradle` to reference a version of the `intellij-kubernetes` plugin that's using the `kubernetes-client 6.4.0`. That version is currently in main branch but it's not published to marketplace yet. So one has to build it locally and reference it in `build.gradle` with it's path (replace `'com.redhat.devtools.intellij.kubernetes:0.6.2.179'` by `<local path where you checked intellij-kubernetes out>/intellij-kubernetes/build/idea-sandbox/plugins/Kubernetes by Red Hat`)

**Comment by adietish** [comment]
The build on `MacOS` fails though with the following:
```
2023-01-26 17:19:22,377 [    234]  ERROR - oncurrency.BoundedTaskExecutor - Resource not found: /idea/nullApplicationInfo.xml
```

This seems caused by the intellij gradle plugin as found earlier here: https://github.com/redhat-developer/intellij-tekton/pull/615#issuecomment-1276426859

bumping it to 1.12.0 (latest) doesnt fix it for me.

File: build.gradle [hunk 2]
```diff
@@ -14,16 +21,16 @@ repositories {
...
+targetCompatibility = '11'
 
 def versionsMap = ['IU-2021.1':'211.6693.108', 'IU-2021.2':'212.4746.57', 'IU-2021.3':'213.5744.122', 'IU-2022.1':'221.5080.1', 'IU-2022.2':'222.3345.16', 'IU-2022.3':'223.7571.175']
 
 intellij {
     version = ideaVersion //for a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
     pluginName = 'com.redhat.devtools.intellij.tekton'
     // use '<git repo clone>/build/idea-sandbox/plugins/Telemetry by Red Hat' if working from source
-    plugins = ['terminal', 'yaml', 'markdown', 'com.intellij.kubernetes:' + versionsMap[ideaVersion], "com.redhat.devtools.intellij.telemetry:0.0.3.33", 'com.redhat.devtools.intellij.kubernetes:0.6.2.179']
+    plugins = ['terminal', 'yaml', 'markdown', "com.redhat.devtools.intellij.telemetry:0.0.3.33", 'com.intellij.kubernetes:' + versionsMap[ideaVersion], '/Users/andredietisheim/Documents/jboss-workspaces/intellij-kubernetes/build/idea-sandbox/plugins/Kubernetes by Red Hat']
```
Comment (adietish) [inline build.gradle:33]: replace `'/Users/andredietisheim/Documents/jboss-workspaces/intellij-kubernetes/build/idea-sandbox/plugins/Kubernetes by Red Hat'` by the intellij-kubernetes plugin release once it's published.

**Comment by adietish** [comment]
@lstocchi can you please test if this works? To build it you need to build `intellij-kubernetes` locally and correct the path in 'plugins = [` so that it points to your locally built plugin.

**Review by lstocchi — COMMENTED** [review]
Tested with almost all tekton actions. Everything works fine. GJ!!
Cannot approve my own PR but LGTM

**Comment by adietish** [comment]
@lstocchi awesomeness, thanks a lot!

**Comment by adietish** [comment]
I also bumped intelij-common to 1.9.0 (Final)

**Pull request merged**